ddf-rpc-api/src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java

```diff
@@ -434,7 +434,7 @@ private Object getSortBy(Map<String, Object> rawSortPolicy) {
 
       Map<String, Object> paramProperties = (Map<String, Object>) params.get("properties");
 
-      if (properties.containsKey("additional-sort-bys")) {
+      if (paramProperties.containsKey("additional-sort-bys")) {
         SortBy[] additionalSortBys =
             (SortBy[])
                 ((List) properties.get("additional-sort-bys"))
```

Comment:
I think this variable needs to be changed too.

Reply:
Absolutely correct.